content/models.py

```diff
@@ -27,3 +27,17 @@ class OurStoryTitle(models.Model):
 
     def __str__(self):
         return self.title
+
+class ChildrensProgram(models.Model):
+	pagetitle = models.CharField(blank=True, null=True, max_length=100)
```

Comment:
I understand now.

My suggestion is to remove the subtitle field and just use the pagetitle field for all of them.